hack/allocate.sh

```diff
@@ -25,13 +25,13 @@ export TERM="${TERM:-dumb}"
 
 main() {
 
-  local knative_serving_version=v1.10.1
-  local knative_eventing_version=v1.10.1
-  local contour_version=v1.10.0
+  local knative_serving_version=v1.13.0
+  local knative_eventing_version=v1.13.0
+  local contour_version=v1.13.0
```

Comment:
Not sure if there's any benefit to test with Contour as networking-layer for Serving. I'd switch to Kourier if possible. In a follow-up PR.